### PR TITLE
Changelog for v1.12.0-rc1

### PR DESCRIPTION
**Pull request by lafriks**
As title :)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,301 @@ This changelog goes through all the changes that have been made in each release
...
+  * Protect default branch against deletion (#11115)
+  * Add X-Total-Count on /repos/{owner]/{repo}/pulls API endpoint (#11113)
+  * Fix status label on branches list vertical alignment (#11109)
+  * Add single release page and latest redirect (#11102)
+  * Add missing commit states to PR checks template (#11085)
+  * Change icon on title for merged PR to git-merge (#11064)
+  * Add MergePull comment type instead of close for merge PR (#11058)
+  * Upgrade jQuery to 3.5.0, remove jQuery-Migrate, fix deprecations (#11055)
+  * Consolidate author name across timeline (#11053)
+  * Refactor UpdateOAuth2Application (#11034)
```
Comment (mrsdizzie) [inline CHANGELOG.md:159]: #11032 is enhancement imo 🙃

Comment (lafriks) [reply CHANGELOG.md:159]: it had no milestone :P

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,301 @@ This changelog goes through all the changes that have been made in each release
...
+## [1.12.0-rc1](https://github.com/go-gitea/gitea/releases/tag/v1.12.0-rc1) - 2020-05-17
+
+* BREAKING
+  * Remove migration support from versions earlier than 1.6.0 (#10026)
+  * Enable ENABLE_HARD_LINE_BREAK by default for rendering markdown (#11162)
+  * Fix sanitizer config - multiple rules (#11133)
+  * [API] When using API CreateRelease set created_unix to the tag commit time (#11218)
+  * [API] Remove check on username if AccessToken authentication (#11015)
+  * [API] Contents API should return 404 on not exist (#10323)
+  * [API] Check Notify (always return json) (#10059)
```
Comment (techknowlogick) [inline CHANGELOG.md]:
```suggestion
  * Check Notify API endpoint always return json (#10059)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,301 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.12.0-rc1](https://github.com/go-gitea/gitea/releases/tag/v1.12.0-rc1) - 2020-05-17
+
+* BREAKING
+  * Remove migration support from versions earlier than 1.6.0 (#10026)
+  * Enable ENABLE_HARD_LINE_BREAK by default for rendering markdown (#11162)
+  * Fix sanitizer config - multiple rules (#11133)
+  * [API] When using API CreateRelease set created_unix to the tag commit time (#11218)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
  * When using API CreateRelease set created_unix to the tag commit time (#11218)
```
#11218

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,302 @@ This changelog goes through all the changes that have been made in each release
...
+  * [API] Remove check on username if AccessToken authentication (#11015)
+  * [API] Contents API should return 404 on not exist (#10323)
```
Comment (techknowlogick) [inline CHANGELOG.md]:
```suggestion
  * Contents API endpoint returns 404 on file not exist (#10323)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,301 @@ This changelog goes through all the changes that have been made in each release
...
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.12.0-rc1](https://github.com/go-gitea/gitea/releases/tag/v1.12.0-rc1) - 2020-05-17
+
+* BREAKING
+  * Remove migration support from versions earlier than 1.6.0 (#10026)
+  * Enable ENABLE_HARD_LINE_BREAK by default for rendering markdown (#11162)
+  * Fix sanitizer config - multiple rules (#11133)
+  * [API] When using API CreateRelease set created_unix to the tag commit time (#11218)
+  * [API] Remove check on username if AccessToken authentication (#11015)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
  * Remove check on username when using AccessToken authentication for the API (#11015)
```
#11015

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,302 @@ This changelog goes through all the changes that have been made in each release
...
+  * Update snapcraft.yaml for core18 and latest features (#11300)
+  * Update JS dependencies (#11297)
+  * Update JS dependencies, min Node.js version 10.13 (#11246)
+  * API: Add pull review endpoints (#11224)
+  * Change default charset for MySQL on install to utf8mb4 (#10989)
+  * Add issue subscription check to API (#10967)
+  * Return issue subscription status from API subscribe (#10966)
+  * API to get single commit via SHA and Ref (#10915)
+  * Fix queue log param (#10733)
+  * update vendor keybase/go-crypto (#10234)
```
Comment (techknowlogick) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md (hunk 3 above)
Comment (lafriks) [reply CHANGELOG.md]: I wanted to specifically tag API breaking changes

File: CHANGELOG.md [hunk 7]
```diff
@@ -4,6 +4,302 @@ This changelog goes through all the changes that have been made in each release
...
+  * Move CSS build to webpack (#9983)
+  * Move fomantic target, update 'make help' (#9945)
+  * Add css extraction and minification to webpack (#9944)
+  * Misc webpack tweaks (#9924)
+  * Make node_modules a order-only prerequisite (#9923)
+  * Update documentation for the go module era (#9751)
+  * Move swagger-ui to webpack/npm and update it to 3.24.3 (#9714)
+  * Use npm to manage fomantic and only build needed components (#9561)
+* MISC
+  * Update JS dependencies (#11412)
```
Comment (techknowlogick) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -4,6 +4,302 @@ This changelog goes through all the changes that have been made in each release
...
+  * Add gnupg to Dockerfile (#11365)
+  * [Vendor] mssqldb: 2019-11-28 -> 2020-04-28 (#11364)
+  * [Vendor] blevesearch v0.8.1 -> v1.0.7 (#11360)
+  * [Vendor] github.com/issue9/identicon: untagged -> v1.0.1 (#11359)
+  * [Vendor] github.com/pquerna/otp:  untagged -> v1.2.0 (#11358)
+  * [Vendor] openid-go: untagged -> v1.0.0 (#11357)
+  * Update snapcraft.yaml for core18 and latest features (#11300)
+  * Update JS dependencies (#11297)
```
Comment (techknowlogick) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -4,6 +4,301 @@ This changelog goes through all the changes that have been made in each release
...
 
+## [1.12.0-rc1](https://github.com/go-gitea/gitea/releases/tag/v1.12.0-rc1) - 2020-05-17
+
+* BREAKING
+  * Remove migration support from versions earlier than 1.6.0 (#10026)
+  * Enable ENABLE_HARD_LINE_BREAK by default for rendering markdown (#11162)
+  * Fix sanitizer config - multiple rules (#11133)
+  * [API] When using API CreateRelease set created_unix to the tag commit time (#11218)
+  * [API] Remove check on username if AccessToken authentication (#11015)
+  * [API] Contents API should return 404 on not exist (#10323)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
  * Return 404 from Contents API when items don't exist (#10323)
```
#10323

File: CHANGELOG.md (hunk 2 above)
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
  * Notification API should always return a JSON object with the current count of notifications (#10059)
```
#10059

File: CHANGELOG.md [hunk 10]
```diff
@@ -4,6 +4,302 @@ This changelog goes through all the changes that have been made in each release
...
 
+## [1.12.0-rc1](https://github.com/go-gitea/gitea/releases/tag/v1.12.0-rc1) - 2020-05-17
+
+* BREAKING
+  * Remove migration support from versions earlier than 1.6.0 (#10026)
+  * Enable ENABLE_HARD_LINE_BREAK by default for rendering markdown (#11162)
+  * Fix sanitizer config - multiple rules (#11133)
+  * When using API CreateRelease set created_unix to the tag commit time (#11218)
+  * Remove check on username when using AccessToken authentication for the API (#11015)
+  * Contents API endpoint returns 404 on file not exist (#10323)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
  * Return 404 from Contents API when items don't exist (#10323)
```
#10323

**Comment by lafriks** [comment]
@zeripath @techknowlogick fixed

File: CHANGELOG.md [hunk 11]
```diff
@@ -4,6 +4,299 @@ This changelog goes through all the changes that have been made in each release
...
+  * Add a way to mark Conversation (code comment) resolved (#11037)
+  * Handle yaml frontmatter (#11016)
+  * Cache PullRequest Divergence (#10914)
+  * Make `gitea admin auth list` formatting configurable (#10844)
+  * Add Matrix webhook (#10831)
+  * Add Organization Wide Labels (#10814)
+  * Allow to set protected file patterns for files that can not be changed under no conditions (#10806)
+  * Option to set default branch at repository creation (#10803)
+  * Add request review from specific reviewers feature in pull request (#10756)
+  * NextCloud oauth (#10562)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
  * Add NextCloud oauth (#10562)
```

File: CHANGELOG.md [hunk 12]
```diff
@@ -4,6 +4,299 @@ This changelog goes through all the changes that have been made in each release
...
+  * Queue: Make WorkerPools and Queues flushable (#10001)
+  * Implement "embedded" command to extract static resources (#9982)
+  * API endpoint for repo transfer (#9947)
+  * Make archive prefixing configurable with a global setting (#9943)
+  * Add Unique Queue infrastructure and move TestPullRequests to this (#9856)
+  * Issue/PR Context Popups (#9822)
+  * Add "Update Branch" button to Pull Requests (#9784)
+  * Add require signed commit for protected branch (#9708)
+  * Mark PR reviews as stale at push and allow to dismiss stale approvals (#9532)
+  * [API] Add notification endpoint (#9488)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
  * Add API notification endpoints (#9488)
```

File: CHANGELOG.md [hunk 13]
```diff
@@ -4,6 +4,299 @@ This changelog goes through all the changes that have been made in each release
...
+  * Prevent accidential selection of language stats bar (#10537)
+  * Vendor: unknwon/cae move to taged version (#10521)
+  * Add fluid-icon (#10491)
+  * inform participants on UI too (#10473)
+  * go 1.14 (#10467)
+  * Add max-file-size to LFS (#10463)
+  * [API] enable paggination for ListRepoTags (#10454)
+  * Update JS dependencies (#10450)
+  * Show the username as a fallback on feeds if full name is blank (#10438)
+  * [Refactor] move APIFormat() of Issue and Label to convert package (#10423)
```
Comment (6543) [inline CHANGELOG.md]: dobwe add refactors to the changelog?

File: CHANGELOG.md [hunk 14]
```diff
@@ -4,6 +4,299 @@ This changelog goes through all the changes that have been made in each release
...
+* MISC
+  * Add gnupg to Dockerfile (#11365)
+  * [Vendor] mssqldb: 2019-11-28 -> 2020-04-28 (#11364)
+  * [Vendor] blevesearch v0.8.1 -> v1.0.7 (#11360)
+  * [Vendor] github.com/issue9/identicon: untagged -> v1.0.1 (#11359)
+  * [Vendor] github.com/pquerna/otp:  untagged -> v1.2.0 (#11358)
+  * [Vendor] openid-go: untagged -> v1.0.0 (#11357)
+  * Update snapcraft.yaml for core18 and latest features (#11300)
+  * Update JS dependencies, min Node.js version 10.13 (#11246)
+  * API: Add pull review endpoints (#11224)
```
Comment (6543) [inline CHANGELOG.md]: This is definetly a feature

File: CHANGELOG.md [hunk 15]
```diff
@@ -4,6 +4,299 @@ This changelog goes through all the changes that have been made in each release
...
+  * Change default charset for MySQL on install to utf8mb4 (#10989)
+  * Add issue subscription check to API (#10967)
+  * Return issue subscription status from API subscribe (#10966)
+  * API to get single commit via SHA and Ref (#10915)
+  * Fix queue log param (#10733)
+  * Add warning when using relative path to app.ini (#10104)
+  * [API] add endpoint to check notifications [Extend #9488] (#9595)
+  * [API] add GET /orgs endpoint (#9560)
+  * [API] add comments endpoint for single comment (#9494)
```
Comment (6543) [inline CHANGELOG.md:296]: Was reverted

File: CHANGELOG.md [hunk 16]
```diff
@@ -4,6 +4,299 @@ This changelog goes through all the changes that have been made in each release
...
+  * When using API CreateRelease set created_unix to the tag commit time (#11218)
+  * Remove check on username when using AccessToken authentication for the API (#11015)
+  * Return 404 from Contents API when items don't exist (#10323)
+  * Notification API should always return a JSON object with the current count of notifications (#10059)
+* FEATURES
+  * Improve config logging when WrappedQueue times out (#11174)
+  * Add branch delete to API (#11112)
+  * Use markdown frontmatter to provide Table of contents, language and frontmatter rendering (#11047)
+  * Add a way to mark Conversation (code comment) resolved (#11037)
+  * Handle yaml frontmatter (#11016)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
  * Handle yaml frontmatter in markdown (#11016)
```

File: CHANGELOG.md [hunk 17]
```diff
@@ -4,6 +4,299 @@ This changelog goes through all the changes that have been made in each release
...
+  * Add Matrix webhook (#10831)
+  * Add Organization Wide Labels (#10814)
+  * Allow to set protected file patterns for files that can not be changed under no conditions (#10806)
+  * Option to set default branch at repository creation (#10803)
+  * Add request review from specific reviewers feature in pull request (#10756)
+  * NextCloud oauth (#10562)
+  * System-wide webhooks (#10546)
+  * Relax sanitization as per https://github.com/jch/html-pipeline (#10527)
+  * Use media links for img in post-process (#10515)
+  * Adds API endpoints to manage OAuth2 Application (list/create/delete) (#10437)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
  * Add API endpoints to manage OAuth2 Application (list/create/delete) (#10437)
```
#10437

File: CHANGELOG.md [hunk 18]
```diff
@@ -4,6 +4,297 @@ This changelog goes through all the changes that have been made in each release
...
+  * Add NextCloud oauth (#10562)
+  * System-wide webhooks (#10546)
+  * Relax sanitization as per https://github.com/jch/html-pipeline (#10527)
+  * Use media links for img in post-process (#10515)
+  * Add API endpoints to manage OAuth2 Application (list/create/delete) (#10437)
+  * Render READMEs in docs/ .gitea or .github from root (#10361)
+  * Add feishu webhook support (#10229)
+  * Cache last commit to accelerate the repository directory page visit (#10069)
+  * Implement basic app.ini and path checks to doctor cmd (#10064)
+  * Queue: Make WorkerPools and Queues flushable (#10001)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
  * Make WorkerPools and Queues flushable (#10001)
```
#10001

File: CHANGELOG.md [hunk 19]
```diff
@@ -4,6 +4,297 @@ This changelog goes through all the changes that have been made in each release
...
+  * Implement "embedded" command to extract static resources (#9982)
+  * API endpoint for repo transfer (#9947)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
  * Add API endpoint for repo transfer (#9947)
```
#9947

File: CHANGELOG.md [hunk 20]
```diff
@@ -4,6 +4,297 @@ This changelog goes through all the changes that have been made in each release
...
+  * Add support for migrating from Gitlab (#9084)
+  * Add support for database schema in PostgreSQL (#8819)
+  * Add setting to set default and global disabled repository units. (#8788)
+  * Language statistics bar for repositories (#8037)
+  * Restricted users (#6274)
+* BUGFIXES
+  * Check branch protection on IsUserAllowedToUpdate (#11448)
+  * Fix margin on attached segment headers when they are separated by other element (#11425)
+  * Fix webhook template when validation errors occur (#11421)
+  * Unfortunately go template if does not shortcut (#11392)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
  * Fix NPE in due to missing signing key on commit page (#11392)
```
#11392

**Comment by lafriks** [comment]
@zeripath @techknowlogick @6543 fixed

File: CHANGELOG.md [hunk 21]
```diff
@@ -4,6 +4,297 @@ This changelog goes through all the changes that have been made in each release
...
+  * Restore active background to Register button on Register page (#11390)
+  * Move LFS directory creation out of NewContext (#11362)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
  * Fix hook failure due to relative LFS_CONTENT_PATH (#11362)
```
#11362

File: CHANGELOG.md [hunk 22]
```diff
@@ -4,6 +4,297 @@ This changelog goes through all the changes that have been made in each release
...
+  * Fix creation of Organization repos by Users with max created personal repos (#11183)
+  * Add option to increase provided OAuth2 token maximum size (#11180)
+  * Log the indexer path on failure (#11172)
+  * Ensure that relative paths in edit preview work (#11143)
+  * Make API EditIssue and EditPullRequest issue notifications (#11123)
+  * Send 404 immediately for known public requests (#11117)
+  * Remove nil inserts in models (#11096)
+  * Add GetReviews() to RetryDownloader (#11093)
+  * Remove nonexistent serviceworker entries (#11091)
+  * ReviewCount: GetApprovalCounts  func sorted wrong (#11086)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
  * Simplify and fix GetApprovalCounts (#11086)
```
#11086

File: CHANGELOG.md [hunk 23]
```diff
@@ -4,6 +4,297 @@ This changelog goes through all the changes that have been made in each release
...
+  * log.Fatal on failure to listen to SSH port (#10795)
+  * Fix forked repo has no icon and language stat. (#10791)
+  * Fix tag/release deletion (#10663)
+  * Fix webhook migration (#10641)
+  * Migration for deleting orphaned dependencies (#10617)
+  * Add migration to fix the old broken merge-bases (#10604)
+  * Update templates for Go 1.14 (#10596)
+  * Remove unnecessary parentheses in wiki/view template (#10583)
+  * Change default value of DefaultCommandExecutionTimeout to match docs (#10581)
+  * Vendor: update github.com/yuin/goldmark to v1.1.24 (#10551)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 24]
```diff
@@ -4,6 +4,297 @@ This changelog goes through all the changes that have been made in each release
...
+  * Better styling for code review comment form (#11413)
+  * Change install description on homepage (#11395)
+  * Ensure search action button is coalesced to adjacent input (#11385)
+  * Switch code editor to Monaco (#11366)
+  * Add paging and archive/private repository filtering to dashboard list (#11321)
+  * Changed image of openid-connect logo for better look on arc-green theme (#11312)
+  * Load Repo Topics on blame view too (#11307)
+  * Change the style in admin notice content view from <p> to <pre> (#11301)
+  * Allow log.xxx.default to set logging settings for the default logger only (#11292)
+  * Update lunny/levelqueue to 0.3.0 (#11285)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
  * Automatically attempt auto recovery of broken disk queues (Update lunny/levelqueue to 0.3.0) (#11285)
```

File: CHANGELOG.md [hunk 25]
```diff
@@ -4,6 +4,297 @@ This changelog goes through all the changes that have been made in each release
...
+  * Change review status icons on pr view style to github style (#10737)
+  * Make pagination optional for API list notification endpoints (#10714)
+  * Fix tab indentation in code view (#10671)
+  * Fix task-list checkbox styling (#10668)
+  * Multiple LFS improvements (#10667)
+  * Make PR message on pushes configurable (#10664)
+  * Move dropzone.js to npm/webpack (#10645)
+  * Ensure Update button is enabled even when CI has failed (#10640)
+  * Add restricted user filter to LDAP authentication (#10600)
+  * OAuth2: add Yandex provider (#8335) (#10564)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
  * Add Yandex OAuth2 provider (#8335) (#10564)
```

File: CHANGELOG.md [hunk 26]
```diff
@@ -4,6 +4,297 @@ This changelog goes through all the changes that have been made in each release
...
+  * Make avatar lookup occur at image request (#10540)
+  * Prevent accidential selection of language stats bar (#10537)
+  * Vendor: unknwon/cae move to taged version (#10521)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 27]
```diff
@@ -4,6 +4,297 @@ This changelog goes through all the changes that have been made in each release
...
+  * Add fluid-icon (#10491)
+  * inform participants on UI too (#10473)
+  * go 1.14 (#10467)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
  * Build with go 1.14 (and raise minimum go version to 1.12) (#10467)
```
#10467

File: CHANGELOG.md [hunk 28]
```diff
@@ -4,6 +4,297 @@ This changelog goes through all the changes that have been made in each release
...
+  * Remove loadCSS and preload woff2 icon fonts (#9976)
+  * Fix commit view JS features, reimplement folding (#9968)
+  * Fix review avatar image (#9962)
+  * Improve notification pager (#9821)
+  * Move jquery and jquery-migrate to npm/webpack (#9813)
+  * Change font to Roboto to support more charsets (#9803)
+  * Move mailer to use a queue (#9789)
+  * Issue search on my related repositories (#9758)
+  * Add "before" query to ListIssueComments and ListRepoIssueComments API (#9685)
+  * Update js dependencies (#9676)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
```
#9676

File: CHANGELOG.md [hunk 29]
```diff
@@ -4,6 +4,297 @@ This changelog goes through all the changes that have been made in each release
...
+  * Move fomantic target, update 'make help' (#9945)
+  * Add css extraction and minification to webpack (#9944)
+  * Misc webpack tweaks (#9924)
+  * Make node_modules a order-only prerequisite (#9923)
+  * Update documentation for the go module era (#9751)
+  * Move swagger-ui to webpack/npm and update it to 3.24.3 (#9714)
+  * Use npm to manage fomantic and only build needed components (#9561)
+* MISC
+  * Add gnupg to Dockerfile (#11365)
+  * [Vendor] mssqldb: 2019-11-28 -> 2020-04-28 (#11364)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
```
#11364

File: CHANGELOG.md [hunk 30]
```diff
@@ -4,6 +4,297 @@ This changelog goes through all the changes that have been made in each release
...
+  * [Vendor] blevesearch v0.8.1 -> v1.0.7 (#11360)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
```
#11360

File: CHANGELOG.md [hunk 31]
```diff
@@ -4,6 +4,297 @@ This changelog goes through all the changes that have been made in each release
...
+  * [Vendor] github.com/issue9/identicon: untagged -> v1.0.1 (#11359)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
```
#11359

File: CHANGELOG.md [hunk 32]
```diff
@@ -4,6 +4,297 @@ This changelog goes through all the changes that have been made in each release
...
+  * [Vendor] github.com/pquerna/otp:  untagged -> v1.2.0 (#11358)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
```
#11358

File: CHANGELOG.md [hunk 33]
```diff
@@ -4,6 +4,297 @@ This changelog goes through all the changes that have been made in each release
...
+  * [Vendor] openid-go: untagged -> v1.0.0 (#11357)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
```
#11357

File: CHANGELOG.md [hunk 34]
```diff
@@ -4,6 +4,297 @@ This changelog goes through all the changes that have been made in each release
...
+  * Ensure search action button is coalesced to adjacent input (#11385)
+  * Switch code editor to Monaco (#11366)
+  * Add paging and archive/private repository filtering to dashboard list (#11321)
+  * Changed image of openid-connect logo for better look on arc-green theme (#11312)
+  * Load Repo Topics on blame view too (#11307)
+  * Change the style in admin notice content view from <p> to <pre> (#11301)
+  * Allow log.xxx.default to set logging settings for the default logger only (#11292)
+  * Update lunny/levelqueue to 0.3.0 (#11285)
+  * Make sendmail a Process and have default timeout (#11256)
+  * cmd: dump: check value of skip-repository flag (#11254)
```
Comment (techknowlogick) [inline CHANGELOG.md]:
```suggestion
  * Check value of skip-repository flag in dump command (#11254)
```

**Comment by lafriks** [comment]
Fixed once more :sweat_smile:

**Review by 6543 — APPROVED** [review]

File: CHANGELOG.md [hunk 35]
```diff
@@ -4,6 +4,289 @@ This changelog goes through all the changes that have been made in each release
...
+  * Make API EditIssue and EditPullRequest issue notifications (#11123)
+  * Send 404 immediately for known public requests (#11117)
+  * Remove nil inserts in models (#11096)
+  * Add GetReviews() to RetryDownloader (#11093)
+  * Remove nonexistent serviceworker entries (#11091)
+  * Simplify and fix GetApprovalCounts (#11086)
+  * Fix wiki revision template and simplify some tmpl conditions (#11080)
+  * Make branch parameter optional for /api/v1/repos/{owner}/{repo}/contents/{filepath} (#11067)
+  * Align review-item svg octicons (#11065)
+  * Fix language label vertical alignment (#11061)
```
Comment (guillep2k) [inline CHANGELOG.md]:
```suggestion
```
Maybe this one should be `skip-log`

File: CHANGELOG.md [hunk 36]
```diff
@@ -4,6 +4,289 @@ This changelog goes through all the changes that have been made in each release
...
+  * Fixed wrong AppSubUrl in multiple templates (#10447)
+  * Fix profile page CSS (#10406)
+  * Inject SVG sprite via ajax (#10320)
+  * Fix migration information update bug when linked github account (#10310)
+  * Allow admin to check org membership by API for other users (#10201)
+  * Fix topics dropdown (#10167)
+  * Ensure DeleteUser is not allowed to Delete Orgs and visa versa (#10134)
+  * Fix IsErrPullClosed (#10093)
+  * Accept punctuation after simple+cross repository issue references (#10091)
+  * More expansions in template repositories (#10021)
```
Comment (guillep2k) [inline CHANGELOG.md]: This one is both fix & enhancement, but I'd say it's more of an enhancement.

File: CHANGELOG.md [hunk 37]
```diff
@@ -4,6 +4,289 @@ This changelog goes through all the changes that have been made in each release
...
+  * Add single release page and latest redirect (#11102)
+  * Add missing commit states to PR checks template (#11085)
+  * Change icon on title for merged PR to git-merge (#11064)
+  * Add MergePull comment type instead of close for merge PR (#11058)
+  * Upgrade jQuery to 3.5.0, remove jQuery-Migrate, fix deprecations (#11055)
+  * Consolidate author name across timeline (#11053)
+  * Refactor UpdateOAuth2Application (#11034)
+  * Support unicode emojis and remove emojify.js (#11032)
+  * Add git hook "warning" to admin panel (#11030)
+  * Add flash notify for email preference seting success (#11027)
```
Comment (guillep2k) [inline CHANGELOG.md]:
```suggestion
  * Add flash notify for email preference setting success (#11027)
```

File: CHANGELOG.md [hunk 38]
```diff
@@ -4,6 +4,289 @@ This changelog goes through all the changes that have been made in each release
...
+  * Add issue subscription check to API (#10967)
+  * Use AJAX for notifications table (#10961)
+  * Adjust label padding (#10957)
+  * Avoiding directory execution on hook (#10954) (#10955)
+  * Migrate ActivityHeatmap to Vue SFC (#10953)
+  * Change merge strategy： do not check write access if user in merge white list (#10951)
+  * Enable GO111MODULE=on globally in Makefile (#10939)
+  * API endpoint to get single commit via SHA and Ref (#10915)
+  * Add accordion to release list and hide non-latest (#10910)
+  * Fix scrollbar issues in dropdowns (#10897)
```
Comment (guillep2k) [inline CHANGELOG.md]: Maybe this is a bugfix?

File: CHANGELOG.md [hunk 39]
```diff
@@ -4,6 +4,289 @@ This changelog goes through all the changes that have been made in each release
...
+  * Add GET /orgs API endpoint (#9560)
+  * API add/generalize pagination (#9452)
+  * Make create org repo API call same as github (#9186)
+* BUILD
+  * Turn off go modules for xgo and gxz (#10963)
+  * Add gitea-vet (#10948)
+  * Rename scripts to build and add revive command as a new build tool command (#10942)
+  * Add 'make lint', restructure 'compliance' pipeline (#10861)
+  * Move JS build dependencies to 'dependencies' (#10763)
+  * Workaround fomantic build issue (#10655)
```
Comment (guillep2k) [inline CHANGELOG.md]:
```suggestion
```
Maybe this should also be skipped.

File: CHANGELOG.md [hunk 40]
```diff
@@ -4,6 +4,289 @@ This changelog goes through all the changes that have been made in each release
...
+  * Use whitelist to find go files, run find only once (#10594)
+  * Frontend build improvements, hopefully fix Fomantic build (#10576)
```
Comment (guillep2k) [inline CHANGELOG.md]:
```suggestion
```
This as well.

File: CHANGELOG.md [hunk 41]
```diff
@@ -4,6 +4,289 @@ This changelog goes through all the changes that have been made in each release
...
+  * Move vue and vue-calendar-heatmap to npm/webpack (#10188)
+  * Move jquery.are-you-sure to npm/webpack (#10063)
+  * Move highlight.js to npm/webpack (#10011)
+  * Generate Bindata iff TAGS="bindata" and not up-to-date (#10004)
```
Comment (guillep2k) [inline CHANGELOG.md]:
```suggestion
  * Generate Bindata if TAGS="bindata" and not up-to-date (#10004)
```

**Review by zeripath — APPROVED** [review]

**Review by guillep2k — APPROVED** [review]

**Pull request merged**